test/functional/sqlsrv/sqlsrv_batch_query.phpt

```diff
@@ -7,7 +7,7 @@ returned otherwise. For information on the expected behaviour of cursors
 with batch queries, see
 https://docs.microsoft.com/en-us/previous-versions/visualstudio/aa266531(v=vs.60)
 --SKIPIF--
-<?php require('skipif_versions_old.inc'); ?>
+<?php require('skipif.inc'); ?>
 --FILE--
```

Comment:
Why this change? Should use the AE skipif

Reply:
Hmm, not sure what happened there. Will fix.